### ISSUE TITLE
<Android>: <Connecting... never ends >

### ISSUE THREAD

**Issue by Kabron287** [analogdevicesinc/scopy#1345]
**Environment:**
 - OS: [Android]
 - FW Version 11
 - SW Version or git tag

**Describe the bug**
Device AlldoCube Kpad
Adalm2000 recognised but Connecting... never ends 

**Screenshots**

![Scopy](https://user-images.githubusercontent.com/2493191/171348461-2386f960-0ea5-49a0-9bfc-2668b30836e5.jpg)

**Comment by Kabron287** [comment]
Thanks for answer, **adisuciu**
Yes, I did it several tens times with disconnected inputs and with external power. If I use Xiaomi MI8 everything works fine, so it is KPad specific.

**Comment by adisuciu** [comment]
We have a new version in the android store. You could give it a try, even though we didn't do anything directly related to your issue. Closing this as it seems it is device specific and we have no means of testing right now.

**Comment by Kabron287** [comment]
Thanks, but no luck.

**Comment by rgetz** [comment]
Did you try a powered hub? I had to use one of those on a Samsung tablet....

- Robin

**Comment by rgetz** [comment]
@Kabron287 is the the tablet you are using?

https://www.amazon.com/ALLDOCUBE-Octa-Core-Processor-2000x1200-Bluetooth/dp/B09W9BLVYQ

@adisuciu if that is the one - it is less than the price of the M2k, can you buy it and test it? 😀 

-Robin

**Comment by Kabron287** [comment]
> @Kabron287это планшет, который вы используете?

Exactly this

**Comment by Kabron287** [comment]
> Пробовали концентратор с питанием? Мне пришлось использовать один из них на планшете Samsung ....

USB tester see no problems

**Comment by adisuciu** [comment]
Before we do any of that, there are a few things to mention.

Since 1.4.0 we fixed a critical bug that caused crashes when the firmware wasn't the latest. The symptoms are not the same (things usually crashed after calibration). We recommend updating the firmware - https://wiki.analog.com/university/tools/pluto/users/firmware?s[]=update&s[]=firmware 

Also uninstalling the app and installing the latest version from the google play store - https://play.google.com/store/apps/details?id=org.adi.scopy&hl=en&gl=US will help a lot.

Running it from the playstore sends back crash reports when crashes occur to us so we can analyze them. We got quite a few crash reports in the play store console, we will analyze them as soon as possible.

If things still go sideways, we can think about testing on specific platforms if that makes sense.
-Adrian
